lib/src/internal/topics_grpc_manager.dart

```diff
@@ -9,11 +9,7 @@ class TopicGrpcManager {
   final _logger = Logger('MomentoTopicClient');
 
   TopicGrpcManager(CredentialProvider credentialProvider) {
-    _channel = ClientChannel(credentialProvider.cacheEndpoint,
-        options: ChannelOptions(
-            keepAlive: ClientKeepAliveOptions(
-                pingInterval: Duration(seconds: 10),
-                timeout: Duration(seconds: 5))));
+    _channel = ClientChannel(credentialProvider.cacheEndpoint);
```

Comment:
removed the keepalive settings as it seemed to interfere with the default retry strategy dart-grpc already implements